content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -431,6 +431,14 @@ Used on: PersistentVolumeClaim
 
 This annotation has been deprecated.
 
+### volume.beta.kubernetes.io/storage-class (deprecated)
+
+Used on: PersistentVolume, PersistentVolumeClaim
+
+This annotation can be used for PersistentVolume(PV) or PersistentVolumeClaim(PVC) to specify the name of [StorageClass](/docs/concepts/storage/storage-classes/).
+
+This annotation has been deprecated.
```

Comment:
Actually this annotation was deprecated in 2017, before v1.9 was released.

I'm not sure adding this back to the docs is useful.

Reply:
From the point of view of the Kubernetes code, this annotation is still in use.

Reply:
We (should) document all annotations that have been used officially any time since the 1.0 release. Even if they are now deprecated. That way, someone who looks can be sure that it's deprecated rather than accidentally left out.

Reply:
@tengqm @mengjiao-liu @sftim Thanks for taking time for the reviewing the changes.

I have noticed though we cover the deprecation in [Using Dynamic Provisioning Page]( https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/#using-dynamic-provisioning). 
In the [Persistent Volumes Page](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class), there is following info:

> In the past, the annotation volume.beta.kubernetes.io/storage-class was used instead of the storageClassName attribute. This annotation is still working; however, it will become fully deprecated in a future Kubernetes release.

Can I please get your thoughts on the info across the above two pages? (If it needs aligning a bit or not)

Reply:
There's some related tidying that's worth doing. Quite separately from docs, we should consider adding a warning any time someone creates an object with that annotation set, or adds that annotation to an existing object that doesn't have it.

_This_ PR should focus on what its title says it will do.

Reply:
Okay. If it is still working, although deprecated several years ago, documenting it here makes sense.